components/app-grid.html

```diff
@@ -0,0 +1,7 @@
+<body class="app-grid">
```

Comment:
I think for the grid to work correctly in your storybook, you need to change the `body`-element to a `div` and add a `footer` 🙂